com.valvesoftware.Steam.yml

```diff
@@ -179,12 +179,12 @@ modules:
         ln -sfr $(readlink -f /app/share/applications/steam.desktop) /app/share/applications/steam.desktop
     sources:
       - type: archive
-        url: http://repo.steampowered.com/steam/archive/precise/steam_1.0.0.69.tar.gz
+        url: https://repo.steampowered.com/steam/archive/beta/steam_1.0.0.69.tar.gz
         sha256: 6f9838014a6b13f953ba726ce7d203946bb93bfd5d074901bc216ad04ab2c767
         x-checker-data:
           type: html
           is-main-source: true
-          url: http://repo.steampowered.com/steam/archive/precise/
+          url: https://repo.steampowered.com/steam/archive/beta/
```

Comment:
Or, if you'd prefer to only be watching for stable releases, s/beta/stable/ throughout this MR.

Reply:
Let's first have stable. In this repo beta isn't a separate track but PR's first go to beta before being landed in stable in a bigger batch so using beta channel would mean it would be used in stable flatpak branch as well.